app/Jobs/NotifyUserOfNewGistComment.php

```diff
@@ -2,13 +2,13 @@
 
 namespace App\Jobs;
 
-use App\NotifiedComment;
 use Carbon\Carbon;
-use Illuminate\Contracts\Queue\ShouldQueue;
-use Illuminate\Queue\InteractsWithQueue;
-use Illuminate\Queue\SerializesModels;
+use App\NotifiedComment;
 use Illuminate\Support\Facades\Log;
 use Illuminate\Support\Facades\Mail;
+use Illuminate\Queue\SerializesModels;
```

Comment:
We sort by alpha, not length

Reply:
Oops, I was using my Framework formatting habits :)